test_conformance/atomics/test_atomics.cpp

```diff
@@ -247,52 +248,50 @@ int test_atomic_function(cl_device_id deviceID, cl_context context,
     log_info("\t(thread count %d, group size %d)\n", (int)threadSize,
              (int)groupSize);
 
-    refValues = (cl_int *)malloc(typeSize * threadSize);
+    refValues.reset(new cl_int[typeSize * threadSize]);
```

Comment:
It would have been nice to get rid of the manual allocations too (and use `std::vector` for example), but at least getting rid of the manual `free`s is an improvement.